available-connectors.json

```diff
@@ -48,12 +48,8 @@
       "port": {
         "type": "number"
       },
-      "user": {
-        "type": "string"
-      },
       "password": {
-        "type": "string",
-        "display": "password"
+        "type": "string"
```

Comment:
It's required (if you don't put add the line, the user entry is not hidden with **** while you type). Reverting and landing.